### PR TITLE
NWPS-1161-right-side-boxes-on-the-publication-page

### PR DESCRIPTION
**Pull request by FranciscoRuizHEE**
Right Hand boxes: Public Info, Alternative Documents, and Languages

### CODE REVIEW

File: repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/report-main.ftl [hunk 1]
```diff
@@ -2,14 +2,40 @@
...
 <@hst.setBundle basename="uk.nhs.hee.web.global"/>
 
+<#macro docDetailBlockForDocLink docLink>
+    <@hst.link var="fileURL" hippobean=docLink>
+        <@hst.param name="forceDownload" value="true"/>
+    </@hst.link>
+    <#assign docType>${docLink.filename?keep_after_last(".")}</#assign>
+    <li>
+        <a class="nhsuk-resources__link" href="${fileURL}" title="${docLink.filename}  (opens in new window)">
+            ${docLink.filename?keep_before_last(".")} - <span class="nhsuk-resources__tag nhsuk-resources__${docType}">${getDocumentFormat(docLink.filename?keep_after_last("."))}</span>
```
Comment (smanivasag) [inline repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/report-main.ftl:16]: Perhaps, we should display a short version of the document format (e.g. ODF, DOCX, etc) like the one indicated in the figma ? Also, the figma includes both long and short version, could we check with Andy/Jon if it was a mistake ? Thanks!

File: repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/report-main.ftl [hunk 2]
```diff
@@ -2,14 +2,40 @@
...
+        </a>
+        <#setting number_format="0.##">
+        <span class="nhsuk-resources__docSize">
+            <#if docLink.lengthMB < 0.3>
```
Comment (smanivasag) [inline repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/report-main.ftl:20]: Do we have a requirement for displaying the file size (as I am struggling to find one in either the ticket/spec) ? Also, I couldn't find the file size display in the right hand side boxes.

File: repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/report-main.ftl [hunk 3]
```diff
@@ -76,6 +102,64 @@
...
+                                            </#if>
+                                        </div>
+                                        <strong>Estimated reading time:</strong> ${landingPage.readTime} min
+                                    </#if>
+                                </div>
+                            </div>
+                            <#if landingPage?? && landingPage.documentVersions?has_content>
+                                <div class="nhsuk-card">
+                                    <div class="nhsuk-card__content">
+                                        <h3 class="nhsuk-heading-m">Alternative versions</h3>
```
Comment (smanivasag) [inline repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/report-main.ftl:135]: Could we handle empty `Alternative versions` boxes (when documents aren't available) ?

Screenshot 2022-12-16 at 08 01 04

File: repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/report-main.ftl [hunk 4]
```diff
@@ -76,6 +102,64 @@
...
+                                                </#if>
+                                            </#list>
+                                        </ul>
+                                    </div>
+                                </div>
+                            </#if>
+                            <#if landingPage?? && landingPage.languageVersions?has_content>
+                                <div class="nhsuk-card">
+                                    <div class="nhsuk-card__content">
+                                        <h3 class="nhsuk-heading-m">Languages</h3>
```
Comment (smanivasag) [inline repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/report-main.ftl:149]: Could we handle empty `Languages` boxes (when documents aren't available) ?

Screenshot 2022-12-16 at 08 01 09

**Review by smanivasag — COMMENTED** [review]
Hi @FranciscoRuizHEE - Thanks for the PR. Could you review my comments please ? Thanks!

**Review by sankar-shunmuga — APPROVED** [review]
thanks @FranciscoRuizHEE @smanivasag for the changes
approved

**Pull request merged**